README.adoc

```diff
@@ -0,0 +1,59 @@
+= LTTng Scope
```

Comment:
How about changing the title to indicate the Alpha status?


# LTTng Scope (Alpha Version)